tbx/static_src/sass/config/_mixins.scss

```diff
@@ -244,11 +244,21 @@
     Navigaiton mixins
 */
 
-@mixin nav-fixed-position($header-height: $header-height-mobile) {
+@mixin nav-fixed-position(
+    $header-height: $header-height-mobile,
+    $full-height: true
+) {
     width: 100%;
-    height: calc(100vh - #{$header-height});
-    // use dvh if supported
-    height: calc(100dvh - #{$header-height});
+    @if $full-height {
+        height: calc(100vh - #{$header-height});
+        // use dvh if supported
+        height: calc(100dvh - #{$header-height});
```

Comment:
Out of interest which browser are we supporting that doesn't support `dvh`?

Reply:
I don't think any officially, but it's still nice to have as a fallback I think. Although I guess the site would look pretty awful without grid...